EIPS/eip-draft-compact-sig.md

```diff
@@ -0,0 +1,180 @@
+---
+eip: 1
+title: EIP Compact Signature Representation
+status: Draft
+type: Informational
+author: Richard Moore <me@ricmoo.com>, Nick Johnson <nick@ethereum.org>
+created: 2019-03-14
+---
+
+## Simply Summary
+
+This proposal describes a compact representation of an Ethereum Signature.
+
+
+## Abstract
+
+The secp256k1 curve permits the computation of the public key of signed
+digest when coupled with a signature, which is used implicitly to
+establish the origin of an EOA-based transaction as well as on-chain in
```

Comment:
Please expand the EOA abbreviation here (externally owned account?)

Reply:
Expanded. :)